tests/Rector/Property/NetteInjectToConstructorInjectionRector/Fixture/parent_with_constructor.php.inc

```diff
@@ -0,0 +1,47 @@
+<?php
+use Nette\Security\Passwords;
+
```

Comment:
The fixture should contain unique psr-4 namespace, like the other, to avoid autoloading 2 same classes with same name and different content.
Check other fixtures for inspiration.